.php-cs-fixer.php

```diff
@@ -0,0 +1,10 @@
+<?php
+$config = \PhpCsFixer\Config::create()
+->setFinder(
+    PhpCsFixer\Finder::create()
+    ->exclude('vendor')
+    ->in(__DIR__)
+);
+// $fixer = new \PhpCsFixer\FixerFactory();
+// $fixer->registerBuiltInFixers()->useRuleSet(new \PhpCsFixer\RuleSet($config->getRules()));
```

Comment:
Are these commented-out lines included intentionally?

Reply:
No, I've messed up a commit somewhere. I'll tidy that and force-push